src/App.js

```diff
@@ -27,42 +29,14 @@ class App extends React.Component {
         <div className="container">
           <p className="text-center mb-4">Browse all the recipes here, select a category or search for an ingredient.</p>
           <div className="row">
-            <div className="col-sm-6 col-lg-4 mb-4">
-              <div className="card">
-                <img src="..." className="card-img-top" alt="..." />
-                <div className="card-body">
-                  <h5 className="card-title">Card title</h5>
-                  <p className="card-text">Some quick example text to build on the card title and make up the bulk of the card’s content.</p>
-                </div>
-              </div>
-            </div>
-            <div className="col-sm-6 col-lg-4 mb-4">
-              <div className="card">
-                <img src="..." className="card-img-top" alt="..." />
-                <div className="card-body">
-                  <h5 className="card-title">Card title</h5>
-                  <p className="card-text">Some quick example text to build on the card title and make up the bulk of the card’s content.</p>
-                </div>
-              </div>
-            </div>
-            <div className="col-sm-6 col-lg-4 mb-4">
-              <div className="card">
-                <img src="..." className="card-img-top" alt="..." />
-                <div className="card-body">
-                  <h5 className="card-title">Card title</h5>
-                  <p className="card-text">Some quick example text to build on the card title and make up the bulk of the card’s content.</p>
-                </div>
-              </div>
-            </div>
-            <div className="col-sm-6 col-lg-4 mb-4">
-              <div className="card">
-                <img src="..." className="card-img-top" alt="..." />
-                <div className="card-body">
-                  <h5 className="card-title">Card title</h5>
-                  <p className="card-text">Some quick example text to build on the card title and make up the bulk of the card’s content.</p>
-                </div>
-              </div>
-            </div>
+            {recipes.map((recipe, i) =>
+              <RecipeCard
+                imageUrl={recipes[i].image}
+                title={recipes[i].title}
+                description={recipes[i].description}
+                key={i}
+              />
```

Comment:
This is ok, as you probably want to see clearly what is used inside the card. Just want to point out that you don't have to pass every attr separately. You could have passed the card object as a whole.  
```js
<RecipeCard 
  reciepe={recipe} 
  key={i} 
/>
```
In this case, if your `RecipeCard` component grows, you don't need to add more and more attrs. If your application grows, you can easily end up with a big amount of attrs. 
For example 👇 
```js
<RecipeCard
  id={recipe.id}
  imageUrl={recipe.image}
  smalImageUrl={recipe.smalImageUrl}
  title={recipe.title}
  subTitle={recipe.subTitle}
  description={recipe.description}
  foo={recipe.foo}
  bar={recipe.bar}
  type={type}
  size={size}
  icon={icon}
  key={i}
/>
```
Note that `type`, `size` and `icon` are attributes for the card. All the rest are nested inside the `recipe` object.
You could easily do 👇 
```js
<RecipeCard
  recipe={recipe}
  type={type}
  size={size}
  icon={icon}
  key={i}
/>
```